### PR TITLE
RDB Shredder

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: .travis/release_relational_database_shredder.yml [hunk 1]
```diff
@@ -15,26 +15,26 @@ targets:
...
     # Optional: Build commands
     build_commands:
-      - cd 3-enrich/scala-hadoop-shred && sbt assembly && cd ../..
+      - cd 4-storage/relational-database-shredder && sbt assembly && cd ../..
 
     # Required: Artifact. Version will be inserted between prefix and suffix
     artifacts:
       - type   : "asis"
-        prefix : "snowplow-hadoop-shred-"
+        prefix : "snowplow-relational-dataase-shredder-"
```
Comment (rgabo) [inline .travis/release_relational_database_shredder.yml]: There's a typo in "dataase" that I just came across.. 😊

File: .travis/release_relational_database_shredder.yml [hunk 2]
```diff
@@ -15,26 +15,26 @@ targets:
...
         suffix : ".jar"
         binary_paths:
-          - "3-enrich/scala-hadoop-shred/target/scala-2.10/snowplow-hadoop-shred-{{ packages.0.build_version }}.jar"
+          - "4-storage/relational-database-shredder/target/scala-2.11/snowplow-relational-datbase-shredder-{{ packages.0.build_version }}.jar"
```
Comment (jbeemster) [inline .travis/release_relational_database_shredder.yml]: Looks like there is one here as well "datbase"

File: .travis/release_relational_database_shredder.yml (hunk 1 above)
Comment (BenFradet) [inline .travis/release_relational_database_shredder.yml]: Thanks for pointing it out :+1:

**Comment by BenFradet** [comment]
closed in favor of #3203

**Pull request closed without merging**